public/preload.js

```diff
@@ -19,6 +19,17 @@ process.once('loaded', () => {
   // Set closed in localStorage, so user does not open in the wallet page
   localStorage.setItem('localstorage:closed', 'true');
 
+  const accessDataRaw = localStorage.getItem('localstorage:accessdata');
+  if (accessDataRaw) {
+    // check if the access data is from a hardware wallet, if so we need to remove it
+    // This ensures that a previously closed hardware wallet will not be loaded again
+    // unless the device is connected
+    const accessData = JSON.parse(accessDataRaw);
+    if ((accessData?.walletFlags & 0x02) > 0) {
```

Comment:
`0x02` is the hardware wallet flag ([ref](https://github.com/HathorNetwork/hathor-wallet-lib/blob/v1.0.4/src/types.ts#L204)).
We can expose `WALLET_FLAGS` from the lib on the next version and import it here but I wish to release this fix asap so I hardcoded the value here for now.

Reply:
Can you create the issue for that, please?

Reply:
Done in https://github.com/HathorNetwork/hathor-wallet-lib/issues/601